src/app/home/modules/schedule/components/schedule-item-detail/schedule-item-detail.component.html

```diff
@@ -35,4 +35,11 @@ <h3>Speakers</h3>
 			</ion-item>
 		</ion-list>
 	</section>
+
+	<section class="schedule-item-detail__link-website">
+		<h3>More details</h3>
+		<p>
+			More details about this session can be found on <a href="TODO" target="_blank" rel="noopener noreferrer">our website</a>.
```

Comment:
TODO!

Reply:
Do we actually want this? Everything that's on the website should also be present in this page? I'm fixing the takeaways at the moment.